react-wasted/src/Foods.js

```diff
@@ -8,7 +8,11 @@ export class Foods extends Component{
         super(props);
         this.state={
             foods:[],
-            createOffer: false
+            createOffer: false,
+            filteredItems:[],
```

Comment:
Is there a purpose for foods now? I think Posting will return only filteredItems anyway. I'm a bit confused about this part

Reply:
Yeah, foods are neccessary now. I haven't thought about this

Reply:
Sorry, do you mean unnecessary?

Reply:
Exactly (((: